mandelgpu.cpp

```diff
@@ -319,29 +319,33 @@ int main(int argc, char* argv[])
     case 'h':
       size_x = 1.0;
       arithmetic_precision = SINGLE;
-      dx = size_x / 
+      dx = size_x /
         static_cast<double>(gl_renderer::instance().get_width());
       break;
     case 'f':
       gl_renderer::instance().toggle_fullscreen(!gl_renderer::instance().is_fullscreen());
       break;
     case 'p':
       // Write image
-      gl_renderer::instance().save_png_screenshot("mandelgpu.png");
+      std::string png1 = "mandelgpu-";
+      std::string pngIndex = std::to_string(std::time(0));
+      std::string png2 = ".png";
+      gl_renderer::instance().save_png_screenshot(png1 + pngIndex + png2); // MARKED HERE ------------------------------------------------------------------
```

Comment:
What is the meaning of the "Marked here" comment? I don't think we need, so can we just remove it?

```suggestion
      gl_renderer::instance().save_png_screenshot(png1 + pngIndex + png2);
```

Reply:
oh yeah, remove that, I marked it there to remember where the line I needed to mess with was